h2o-py/h2o/estimators/random_forest.py

```diff
@@ -627,7 +627,10 @@ def checkpoint(self):
     @checkpoint.setter
     def checkpoint(self, checkpoint):
         assert_is_type(checkpoint, None, str, H2OEstimator)
-        self._parms["checkpoint"] = checkpoint
+        if(issubclass(checkpoint, H2OEstimator)):
```

Comment:
This is enhancement. Makes it possible to insert just the model to be checkpointed and not to explicitely extract its `model_id`.